src/js/plugin/external/PluginDispatcher.js

```diff
@@ -0,0 +1,5 @@
+import {Dispatcher} from "flux";
```

Comment:
Having these files in a directory named "external" kinda feels wrong. If there is need for separation (and I don't feel we necessarily need an extra dir) we should call the dir "shared" or something similar as all these files are also used within Marathon UI itself.